perception/multi_object_tracker/src/tracker/model/bicycle_tracker.cpp

```diff
@@ -132,7 +132,7 @@ BicycleTracker::BicycleTracker(
     bounding_box_ = {
       object.shape.dimensions.x, object.shape.dimensions.y, object.shape.dimensions.z};
   } else {
-    bounding_box_ = {1.0, 0.5, 1.7};
```

Comment:
I am concerning if this default bicycle shape is correct or not. 
I think longitude side should longer in normal bicycle.